From dca71a36ea82b8679d552bfd6179d6a925029ef0 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Thu, 14 Feb 2013 19:29:05 +0100
Subject: [PATCH] Delegate deb(5) format description to the man page

The information in this section is in any case not very complete,
and covered already in way more detail by the man page.
---
 policy.sgml | 18 +-----------------
 1 file changed, 1 insertion(+), 17 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index 39b796b..a027f11 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -10866,25 +10866,9 @@ END-INFO-DIR-ENTRY
       <heading>Binary packages (from old Packaging Manual)</heading>
 
       <p>
-	The binary package has two main sections.  The first part
-	consists of various control information files and scripts used
-	by <prgn>dpkg</prgn> when installing and removing.  See <ref
-	id="pkg-controlarea">.
+	See <manref name="deb" section="5"> and <ref id="pkg-controlarea">.
       </p>
 
-      <p>
-	The second part is an archive containing the files and
-	directories to be installed.
-      </p>
-
-      <p>
-	In the future binary packages may also contain other
-	components, such as checksums and digital signatures. The
-	format for the archive is described in full in the
-	<file>deb(5)</file> man page.
-      </p>
-
-
       <sect id="pkg-bincreating"><heading>Creating package files -
       <prgn>dpkg-deb</prgn>
 	</heading>
-- 
1.8.1.3

